src/cli.ts

```diff
@@ -32,7 +32,9 @@ if (Deno.args.length < 1 || args.help || args.h) {
 
 const url = args._[0];
 let path = args.out;
-const authorization = args.auth;
+const authorizationString = args.auth;
```

Comment:
can we accept a list instead of a single header to make it future-proof? Also, I think we should rename --auth to --headers as the argument. 
Something similar to the below code can we used.
`const requestHeaders: Record<string, string> = {};
    headers.forEach(header => {
        const [key, value] = header.split(':');
        requestHeaders[key.trim()] = value.trim();
    });`

Reply:
Just saw your msg, I can pick this enhancement.